examples/survey-feedback-analysis/main.bal

```diff
@@ -20,32 +20,8 @@ import ballerinax/slack;
 
 configurable string token = ?;
 
-string channelName = "survey-coordination";
-string surveyRequestMessage = "Reply to this survey message to give input on the company";
-
-type MessagesItem record {
-    string text;
-    string thread_ts;
-    string ts;
-    string 'type;
-    string user;
-    string parent_user_id?;
-    string last_read?;
-    int reply_count?;
-    boolean subscribed?;
-    int unread_count?;
-};
-
-type Response_metadata record {
-    string next_cursor;
-};
-
-type RepliesResponse record {
-    boolean has_more;
-    MessagesItem[] messages;
-    boolean ok;
-    Response_metadata response_metadata;
-};
+final string channelName = "survey-coordination";
+final string surveyRequestMessage = "Reply to this survey message to give input on the company";
```

Comment:
```suggestion
const CHANNEL_NAME = "survey-coordination";
const SURVEY_REQUEST_MSG = "Reply to this survey message to give input on the company";
```

Reply:
@Chilliwiddit just applying the suggestion won't work :) you have to update the usages as well

Reply:
Yes Sorry Nipuna it slipped my mind. I just fixed the above usage